logic/src/commonMain/kotlin/com/wire/kalium/logic/data/session/SessionRepository.kt

```diff
@@ -90,7 +90,7 @@ internal class SessionDataSource(
 
     override fun allValidSessions(): Either<StorageFailure, List<AuthSession>> =
         wrapStorageRequest {
-            sessionStorage.allSessions()?.filter { it.value is AuthSessionEntity.Valid }?.values?.toList()
+            sessionStorage.allSessions()?.filter { it.value is AuthSessionEntity.Valid }?.values?.toList()?.ifEmpty { null }
```

Comment:
here is returning null, but the function return is not nullable, am I missing something here?